rom/dev/src/rom_env.rs

```diff
@@ -175,4 +195,9 @@ impl RomEnv {
     pub fn pcr_bank(&self) -> &EnvCell<PcrBank> {
         &self.pcr_bank
     }
+
+    /// Get ICCM Range
+    pub fn iccm_range(&self) -> &Range<u32> {
```

Comment:
FWIW, the compiler would probably have an easier time optimizing this if iccm_range(&self) returned a constant rather than a field.

Reply:
Updated.